natural_language_processing/text_generation/alpaca/run.py

```diff
@@ -20,6 +20,7 @@ def run_single_pass(pytorch_runner, dataset):
 
 
     model = AutoModelForCausalLM.from_pretrained(model_path)
+    model.merge_qkv()
```

Comment:
submodule of transformers at natural_language_processing/text_generation/alpaca/ needs to be updated to _karol/llama-compile-v3_ branch